lib/rucio/core/transfer.py

```diff
@@ -719,10 +720,14 @@ def get_transfer_requests_and_source_replicas(total_workers=0, worker_number=0,
         # In case the source_rse and the dest_rse are connected, the list contains only the destination RSE
         # In case of non-connected, the list contains all the intermediary RSEs
         list_hops = []
+        include_multihop = False
+        if transfertool == 'fts':
```

Comment:
We need to cover the default case as well (`ransfertool=None`)
let's change this to `if transfertool in ['fts', None]:`